src/pudl/metadata/resources/eia860.py

```diff
@@ -476,6 +476,36 @@
         "sources": ["eia860", "eia923"],
         "etl_group": "eia860",
     },
+    #     "emissions_control_equipment_eia860": {
+    #         "description": (
+    #             """
+    # The cost, type, operating status, retirement date, and install year of emissions control
+    # equipment reported to EIA. Includes control ids for sulfur dioxide (SO2), particulate
+    # matter, mercury, nitrogen oxide (NOX), and acid (HCl) gas monitoring.
+    # """
+    #         ),
+    #         "schema": {
+    #             "fields": [
+    #                 "report_year",
+    #                 "plant_id_eia",
+    #                 "emission_control_id_pudl",
+    #                 "data_maturity",
+    #                 "emission_control_equipment_type",
+    #                 "operational_status_code",
+    #                 "mercury_control_id_eia",
+    #                 "nox_control_id_eia",
+    #                 "particulate_control_id_eia",
+    #                 "so2_control_id_eia",
+    #                 "acid_gas_control",
+    #                 "emission_control_equipment_cost",
+    #             ],
+    #             "primary_key": ["report_year", "plant_id_eia", "emission_control_id_pudl"],
+    #             "foreign_key_rules": {},
```

Comment:
I don't really understand how this works @cmgosnell maybe you can help me here?

Reply:
I don't think you want `foreign_key_rules` on this table. The fk rules say: if you find this tables PKs in *other* tables, you should create a FK relationship in that table. You can also `exclude` tables if need be. You can look at the `boilers_eia860` tables metadata as an example of both.

Reply:
Hmmm ok. The primary keys are `report_year`, `plant_id_eia`, and `emission_control_id_pudl`. Are there any circumstances were you would report part of the primary key (i.e., the first two cols) because the last one is made specifically for this table? Or would you only list them if all three were found in another table that you specifically wanted to link to?